skimage/segmentation/tests/test_slic.py

```diff
@@ -538,3 +538,23 @@ def test_dtype_support(dtype):
 
     # Simply run the function to assert that it runs without error
     slic(img, start_label=1)
+
+
+def test_start_label_fix():
+    """Tests the fix for a bug producing a label < start_label (gh-6240).
```

Comment:
```suggestion
    """Test the fix for a bug producing a label < start_label (gh-6240).
```
Sorry for coming late! Thanks for adding this test :smiley: